dependencies.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import pandas as pd
+import json
+
+if __name__ == "__main__":
+    rc = subprocess.call("./dependencies.sh")
+
+    with open("dependencies_analysis/dependencies.json", encoding='utf-8') as file:
+        lst = json.load(file)
+
+    with open("dependencies_analysis/upgrade.json", encoding='utf-8') as file:
+        upg = json.load(file)
+
+    out = []
+    for d in lst:
+        out.append({})
+        for k, v in d["package"].items():
+            out[-1][f"package.{k}"] = v
+        out[-1]["dependencies"] = "; ".join(v["key"] for v in d["dependencies"])
+
+    df_dep = pd.DataFrame(out)
```

Comment:
pandas is a pretty heavyweight way of converting json to a csv, and I don't really want it in the project deps since it's a massive library.

Reply:
Resolved by moving this out of the standard DOAJ install, as discussed this morning, and just running it as an external script to munge the data from the dependencies.sh